apps/deploy-web/src/components/get-started/GetStartedStepper.tsx

```diff
@@ -88,41 +90,57 @@ export const GetStartedStepper: React.FunctionComponent = () => {
           onClick={() => (activeStep > 0 ? onStepClick(0) : null)}
           classes={{ label: cn("text-xl tracking-tight", { ["cursor-pointer hover:text-primary"]: activeStep > 0, ["!font-bold"]: activeStep === 0 }) }}
         >
-          Wallet
+          Billing
         </StepLabel>
 
         <StepContent>
-          <p className="text-muted-foreground">
-            You need at least {minDeposit.akt} AKT or {minDeposit.usdc} USDC in your wallet to deploy on Akash. If you don't have {minDeposit.akt} AKT or{" "}
-            {minDeposit.usdc} USDC, you can switch to the sandbox or ask help in our <ExternalLink href="https://discord.gg/akash" text="Discord" />.
-          </p>
+          {!isWalletConnected && (
```

Comment:
Do we want to hide this behind `NEXT_PUBLIC_BILLING_ENABLED`?

Reply:
totally forgot about this. will have to review this PR changes to make sure nothing leaks

Reply:
ok, pushed an update